R/splice.R

```diff
@@ -25,6 +20,8 @@
 #' c(inputs, arg3 = c("c1", "c2")) %>% str()
 #' @export
 splice <- function(...) {
+  lifecycle::deprecate_warn("0.4.0", "splice()")
```

Comment:
Do we now plan for a 1.0.0 right away?

Reply:
Maaaaaybe. If so I'll do another pass to update 0.4.0 to 1.0.0